.gitignore

```diff
@@ -0,0 +1,8 @@
+*.pyc
+__pycache__
```

Comment:
could you remove this .gitignore file?

Reply:
OK, but I have to ask, why is having a `.gitignore` file an issue? :-)

Reply:
Hi @vloncar Good question. It is not an issue. Just we do not use that and have not encountered any problem without it (yet).